Include/internal/pycore_object.h

```diff
@@ -38,11 +38,12 @@ static inline void _PyObject_GC_TRACK_impl(const char *filename, int lineno,
                           "object is in generation which is garbage collected",
                           filename, lineno, "_PyObject_GC_TRACK");
 
-    PyGC_Head *last = (PyGC_Head*)(_PyRuntime.gc.generation0->_gc_prev);
+    _PyRuntimeState *runtime = _PyRuntimeState_Get();
+    PyGC_Head *last = (PyGC_Head*)(runtime->gc.generation0->_gc_prev);
```

Comment:
IMHO it's wrong to put the gc structure into the runtime: it should be moved to PyInterpreterState instead, and so this function should get the current interpreter, not the runtime.

Reply:
Agreed. :)  See gh-13219.